tchannel/tornado/peer.py

```diff
@@ -96,6 +100,12 @@ def __init__(self, tchannel, hostport):
         # helps avoid making multiple outgoing connections.
         self._connecting = None
 
+        # score is used to measure the performance of the peer.
+        # It will be used in the peer heap.
+        self.score = sys.maxint
+        # index records the position of the peer in the peer heap
```

Comment:
Why is the default score maxint? If anything it should be 0 since you're not connected to the peer initially.